ocs_ci/utility/utils.py

```diff
@@ -2282,6 +2288,113 @@ def create_ceph_health_cmd(namespace):
     return ceph_health_cmd
 
 
+def run_ceph_health_cmd(namespace):
+    """
+    Run the ceph health command
+
+    Args:
+        namespace: Namespace of OCS
+
+    Raises:
+        CommandFailed: In case the rook-ceph-tools pod failed to reach the Ready state.
+
+    Returns:
+        str: The output of the ceph health command
+
+    """
+    wait_for_rook_ceph_tools_pod(namespace)
```

Comment:
Shouldn't this work?
```python
ct_pod = pod.get_ceph_tools_pod()
return = ct_pod.exec_ceph_cmd(ceph_cmd="ceph health", format="plain")
```

Reply:
Yes, it should work. The thing is that it will change the steps of the function `ceph_health_check_base`. Because in the process `ceph_health_check_base`, we use `oc` commands. From my side, we can replace it with the existing function `get_ceph_tools_pod`. The question is if there was a reason for using these `oc` commands, so we will not get regression failures.

Reply:
Didn't get the thing with `oc` commands, can you please elaborate?

Reply:
Yes, I will be more clear. In the original function, we execute the code: 

> run_cmd(
        f"oc wait --for condition=ready pod "
        f"-l app=rook-ceph-tools "
        f"-n {namespace} "
        f"--timeout=300s"
    )
    ceph_health_cmd = create_ceph_health_cmd(namespace)
    run_cmd(ceph_health_cmd)

And now, we will use the function `get_ceph_tools_pod` instead. So, the question is if it will work correctly in all the tests.